.mailmap

```diff
@@ -101,14 +123,18 @@ Petri Lievonen <plievone@cc.hut.fi>
 Petri Lievonen <plievone@cc.hut.fi> <petri.lievonen@tkk.fi>
 Pieter Vanderwerff <me@pieter.io> <pieter@heyday.co.nz>
 Pouja Nikray <poujanik@gmail.com>
+Rainer Oviir <roviir@gmail.com> <raineroviir@rainers-MacBook-Pro.local>
 Ray <ray@tomo.im>
 Richard Feldman <richard.t.feldman@gmail.com> <richard@noredink.com>
 Richard Livesey <Livesey7@hotmail.co.uk>
 Rob Arnold <robarnold@cs.cmu.edu>
+Robin Frischmann <robin@rofrischmann.de>
 Sander Spies <sandermail@gmail.com>
 Scott Feeney <scott@oceanbase.org> <smf@fb.com>
 Sebastian Markbåge <sebastian@calyptus.eu> <sema@fb.com>
 Sergey Rubanov <chi187@gmail.com>
+Shogun Sea <shogunsea08@gmail.com> <xxin@groupon.com>
```

Comment:
Unfortunately no - that's the email your commit was made with so it's in the git history, which is how the authors file is generated. `git shortlog` uses the `.mailmap` file to map so that the `AUTHORS` file has the right final data. ([script](https://github.com/facebook/react/blob/master/scripts/authors)).